airflow/sensors/base.py

```diff
@@ -114,15 +118,27 @@ def _orig_start_date(
     )
 
 
+class SkipPolicy(str, enum.Enum):
+    """Action to TODO."""
+
+    NONE = "none"
+
+    # If poke method raises an exception, sensor will be skipped.
+    SKIP_ON_ANY_ERROR = "skip_on_any_error"
+
+    SKIP_ONLY_SOFT_ERROR = "skip_only_soft_error"
```

Comment:
Namings might need some minor improvements:

```suggestion
    SKIP_ON_SOFT_ERROR = "skip_on_soft_error"
```